pkg/constants/constants.go

```diff
@@ -81,26 +81,26 @@ const (
-	TargetUtilizationPercentage                 = KServeAPIGroupName + "/targetUtilizationPercentage"
-	MinScaleAnnotationKey                       = KnativeAutoscalingAPIGroupName + "/min-scale"
-	MaxScaleAnnotationKey                       = KnativeAutoscalingAPIGroupName + "/max-scale"
-	RollOutDurationAnnotationKey                = KnativeServingAPIGroupName + "/rollout-duration"
-	KnativeOpenshiftEnablePassthroughKey        = "serving.knative.openshift.io/enablePassthrough"
-	EnableMetricAggregation                     = KServeAPIGroupName + "/enable-metric-aggregation"
-	SetPrometheusAnnotation                     = KServeAPIGroupName + "/enable-prometheus-scraping"
-	KserveContainerPrometheusPortKey            = "prometheus.kserve.io/port"
-	KServeContainerPrometheusPathKey            = "prometheus.kserve.io/path"
-	PrometheusPortAnnotationKey                 = "prometheus.io/port"
-	PrometheusPathAnnotationKey                 = "prometheus.io/path"
-	StorageReadonlyAnnotationKey                = "storage.kserve.io/readonly"
-	DefaultPrometheusPath                       = "/metrics"
-	QueueProxyAggregatePrometheusMetricsPort    = 9088
-	DefaultPodPrometheusPort                    = "9091"
+	InferenceServiceGKEAcceleratorAnnotationKey       = KServeAPIGroupName + "/gke-accelerator"
+	DeploymentMode                                    = KServeAPIGroupName + "/deploymentMode"
+	EnableRoutingTagAnnotationKey                     = KServeAPIGroupName + "/enable-tag-routing"
+	AutoscalerClass                                   = KServeAPIGroupName + "/autoscalerClass"
+	AutoscalerMetrics                                 = KServeAPIGroupName + "/metrics"
+	TargetUtilizationPercentage                       = KServeAPIGroupName + "/targetUtilizationPercentage"
+	MinScaleAnnotationKey                             = KnativeAutoscalingAPIGroupName + "/min-scale"
+	MaxScaleAnnotationKey                             = KnativeAutoscalingAPIGroupName + "/max-scale"
+	RollOutDurationAnnotationKey                      = KnativeServingAPIGroupName + "/rollout-duration"
+	KnativeOpenshiftEnablePassthroughKey              = "serving.knative.openshift.io/enablePassthrough"
+	EnableMetricAggregation                           = KServeAPIGroupName + "/enable-metric-aggregation"
+	SetPrometheusAnnotation                           = KServeAPIGroupName + "/enable-prometheus-scraping"
+	KserveContainerPrometheusPortKey                  = "prometheus.kserve.io/port"
+	KServeContainerPrometheusPathKey                  = "prometheus.kserve.io/path"
+	PrometheusPortAnnotationKey                       = "prometheus.io/port"
+	PrometheusPathAnnotationKey                       = "prometheus.io/path"
+	StorageReadonlyAnnotationKey                      = "storage.kserve.io/readonly"
+	DefaultPrometheusPath                             = "/metrics"
+	QueueProxyAggregatePrometheusMetricsPort    int32 = 9088
+	DefaultPodPrometheusPort                          = "9091"
```

Comment:
should we use int here as well?

Reply:
Before it was int. kubernetes defines port as int32. This will lead to downcast and the lint will complain. Even if we change it to either string or int or int32,  the conversion is unavoidable. So i thought it is better to do a upcast rather than downcast and also the linter will be happy.